### DISCUSSION TITLE
where is code of zeus cli?

### DISCUSSION THREAD

**Discussion by xunxky** [zepp-health/zeppos-forum#378] [Developer Feedback]
where is code for https://www.npmjs.com/package/@zeppos/zeus-cli ? 
it has a bug ? 
fix line @zeppos/zeus-cli/private-modules/zeppos-app-utils/dist/modules/create/index.js:141 
watchfaceTemplateChoices = { v1: [], v2: [] ,'v3':[], 'v3.5':[] };

**Comment by zeppos-hx** [comment]
Thanks you! is it in beta channel or stable channel? I will update soon

**Reply by xunxky** [reply]
Actually it is both and probably all since v3 and v3.5 have been added and the `watchfaceTemplateChoices` arrays are accessed without being instantiated. 
/@zeppos/zeus-cli/private-modules/zeppos-app-utils/dist/modules/create/index.js
Line 141 is to fix 
in 
https://www.npmjs.com/package/@zeppos/zeus-cli?activeTab=code 
(https://www.npmjs.com/package/@zeppos/zeus-cli/file/cc6c67e7532b9f558e41cdfae9ff23f314928e344da5be8613f633f9e33b127b)


The error happens at Line 202 where the array is assumed to exist for the `osVersion`

**Reply by zeppos-hx** [reply]
are you find any errors while using it for creating watchface?
zeus does not have the v3 or v3.5 options when creating watchface, so it might be not a bug

**Reply by xunxky** [reply]
I am 100% positive it is a bug! You should try it out on a fresh install. 
I received an error and the CODE screams that it should fail, as it is accessing a function that isn't there.
I fixed the respective line in my modules and it works after that ...

**Reply by zeppos-hx** [reply]
I released [zeus 1.6.7-beta2](https://www.npmjs.com/package/@zeppos/zeus-cli/v/1.6.7-beta2) , added the fix code, thank you
